experimental/check_gpu_ecc/check_gpu_ecc.py

```diff
@@ -9,11 +9,16 @@
 import socket
 import json
 import csv
```

Comment:
Imports should follow [PEP 8](https://peps.python.org/pep-0008/#imports).

Reply:
Please be more specific on what change you want?

Reply:
Just linting with `isort` and removing unused imports results in:

```
import csv
import json
import socket
import subprocess
import sys
from datetime import datetime, timedelta
from urllib.request import Request, urlopen
```

Reply:
Done